superset-frontend/webpack.config.js

```diff
@@ -199,57 +189,29 @@ const config = {
       {
         test: /\.tsx?$/,
         use: [
-          { loader: 'cache-loader' },
-          {
-            loader: 'thread-loader',
-            options: {
-              // there should be 1 cpu for the fork-ts-checker-webpack-plugin
-              workers: os.cpus().length - 1,
-            },
-          },
+          'thread-loader',
+          babelLoader,
```

Comment:
`.tsx?` scripts are now transformed by Babel, too.

Reply:
Actually wanna get rid of `ts-loader` to make it consistent with how the other repos are setup, so we can all be on the same set of transpiler behaviors and constraints.